pkg/api/testapi/testapi.go

```diff
@@ -268,15 +266,6 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
-	if _, ok := Groups[kubeadm.GroupName]; !ok {
```

Comment:
You shouldn't even exist in the same scheme, so the existing round trip tests shouldn't've have picked you up.  In addition, you want a roundtripping test regardless of what the current API registry has registered for versions, so if the fuzzer somehow got wired to `testapi` that was likely a mistake.